examples/README.md

```diff
@@ -0,0 +1,84 @@
+# Standardization of PyTA Examples
+
+
+## Purpose
+
+Examples should illustrate an error in a succinct way, while clearly explaining
+the cause of the error to a beginner audience.
+
+
+## Sorts of Examples
+
+* Custom examples: located in `pyta/examples/`
+* PyLint examples: located in `pyta/examples/pylint/`
+
+
+## How Examples are Included
+
+The examples are included in the website index file with the `pandocfilters`
+module. See `/website/index.md` for the syntax to include the examples. Note
+that further explanation in addition to the example can be added directly to
+the `index.md` file.
+
+
+## Documentation
+
+
+#### Module Docstrings
+
+Examples typically do not use module docstrings.
+
+
+#### Function Docstrings
+
+Function docstrings use double quotes. Try to minimize the vertical size for
```

Comment:
Good question. Use double-quotes for function docstrings and single-quotes otherwise.